frontend/src/components/skills/Dropzone.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box } from '@chakra-ui/react';
+import { FC, ReactNode } from 'react';
+import { useDrop } from 'react-dnd';
+import { Item, ItemType } from '../../types/ItemTypes';
+
+interface Props {
+    onItemDropped?: (item: Item) => void;
+    children?: ReactNode;
+}
+
+interface DropProps {
+    canDrop: boolean;
+}
+
+const Dropzone: FC<Props> = (props) => {
+    const [{ canDrop }, dropRef] = useDrop<Item, unknown, DropProps>(() => ({
```

Comment:
what is the unknown here in ref to?

Reply:
It is the `DropResult`, from what I understand it's an alternative way of getting the item that was dropped using the `monitor.getDropResult` (Instead of using the drop callback). By default, it's `unknown` and I had to set it to something if I wanted to specify the `DropProps` type.